.github/workflows/manual-comps-test.yml

```diff
@@ -48,23 +53,29 @@ jobs:
     runs-on: ubuntu-latest
     outputs:
       services: ${{ steps.get-matrix.outputs.services }}
+      nodes: ${{ steps.get-matrix.outputs.nodes }}
```

Comment:
not all steps need node list, only build image need this. otherwise the test will be duplicated right?